### PR TITLE
Es and fr book

### PR DESCRIPTION
**Pull request by 0x4D5352**
Split out Spanish and French for #1684. Didn't appear to have any of the same indentation issues but I could be a bit tab-blind after all this, lol.

### CODE REVIEW

File: es/book/trabajando_con_tablas.md [hunk 1]
```diff
@@ -189,46 +189,47 @@ Además de seleccionar datos de una tabla, también podemos actualizar lo que co
...
 ```
-> open rustfmt.toml | add next_edition 2021 | save rustfmt2.toml
-/home/sophia/Source/nushell(master)> open rustfmt2.toml
----------+--------------
- edition | next_edition
----------+--------------
- 2018    | 2021
----------+--------------
+open rustfmt.toml | add next_edition 2021 | save rustfmt2.toml
+# /home/sophia/Source/nushell(master)
```
Comment (NotTheDr01ds) [inline es/book/trabajando_con_tablas.md:226]:
```suggestion
```

Leftover prompt line

File: fr/book/cheat_sheet.md [hunk 2]
```diff
@@ -514,9 +512,9 @@ export def main [] {
...
-> use greetings.nu
-> greetings
-greetings and salutations!
-> greetings hello world
-hello world!
+use greetings.nu
+greetings
+# => greetings and salutations!
+# => greetings hello world
+# => hello world!
```
Comment (NotTheDr01ds) [inline fr/book/cheat_sheet.md:519]:
```suggestion
use greetings.nu
greetings
# => greetings and salutations!
greetings hello world
# => hello world!
```

**Review by NotTheDr01ds — CHANGES_REQUESTED** [review]
Thanks! A few more minor tweaks.

I'm also finding out just how outdated some of these translations are - Apparently the Spanish one still includes an `add` command example, which was renamed to `insert` in 2019 apparently!

File: es/book/instalacion.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Descarga el archivo actual `.zip` [de la página de releases](https://github.com
 y posteriormente agrega Nu a la variable de entorno `PATH`. Una vez que hagamos eso, podemos ejecutar Nu usando el comando `nu`:
 
 ```
- > nu
+ nu
  C:\Users\user>
```
[the remark this answers is not in the document]
Comment (0x4D5352) [reply es/book/instalacion.md:23]: Yeah, I wasn't quite sure whether or not to keep those in but they don't seem super relevant.

File: es/book/entorno.md [hunk 4]
```diff
@@ -29,6 +29,6 @@ A common shorthand, inspired by Bash and others, is also available. You can writ
 Una atajo común, inspirada en Bash y otros, también está disponible. Puedes escribir el ejemplo anterior como:
 
 ```
-> MI_VARIABLE=VALOR echo $nu.env.MI_VARIABLE
-VALOR
+# => MI_VARIABLE=VALOR echo $nu.env.MI_VARIABLE
```
Comment (NotTheDr01ds) [inline es/book/entorno.md:32]: Looks like this one slipped through:

```suggestion
MI_VARIABLE=VALOR echo $nu.env.MI_VARIABLE
```

File: es/book/instalacion.md [hunk 5]
```diff
@@ -113,23 +112,24 @@ brew install openssl cmake
...
 
 Para todas las funcionalidades disponibles, la manera más fácil es descargar la fuente de Nu y construírlo usted mismo usando las herramientas de Rust:
 
 ```
-> git clone https://github.com/nushell/nushell.git
-> cd nushell
-nushell> cargo install --path . --locked --force --features=stable
+git clone https://github.com/nushell/nushell.git
+cd nushell
+# ./nushell
```
Comment (NotTheDr01ds) [inline es/book/instalacion.md:131]: Probably not necessary here - I'd probably just remove this one.

File: fr/book/cheat_sheet.md [hunk 6]
```diff
@@ -377,97 +375,97 @@ utiliser le flag comme un switch avec un racourcis (-a) pour l'âge :
...
-Error: nu::parser::expected_keyword
-
-  × Capture of mutable variable.
-   ╭─[entry #83:1:18]
- 1 │ [1 2 3] | each { $x += 1 }
-   ·                  ─┬
-   ·                   ╰── capture of mutable variable
-   ╰────
+mut x = 0
+# => [1 2 3] | each { $x += 1 }
```
Comment (NotTheDr01ds) [inline fr/book/cheat_sheet.md:431]:
```suggestion
[1 2 3] | each { $x += 1 }
```

**Review by NotTheDr01ds — CHANGES_REQUESTED** [review]
Thank you again, and apologies for not getting the reviews on these done sooner.  Just a few things on this one and it will be ready.

File: es/book/instalacion.md (hunk 5 above)
Comment (NotTheDr01ds) [inline es/book/instalacion.md:131]:
```suggestion
```

Comment (NotTheDr01ds) [inline es/book/instalacion.md]:
```suggestion
```

File: es/book/instalacion.md [hunk 7]
```diff
@@ -148,24 +148,25 @@ También podemos contruir nuestro propio Nu directamente desde github. Esto nos
...
 ```
 
 Git nos clonará el repositorio principal de Nu. Partiendo de ahí podemos contruir y arrancar Nu si estamos usando `rustup` con:
 
 ```
 
-> cd nushell
-> nushell> cargo build --workspace --features=stable; cargo run --features=stable
+cd nushell
+# ./nushell
```
Comment (NotTheDr01ds) [inline es/book/instalacion.md:159]:
```suggestion
```

**Review by NotTheDr01ds — APPROVED** [review]

**Pull request merged**